sdk/src/feature_set.rs

```diff
@@ -995,7 +999,8 @@ lazy_static! {
         (enable_tower_sync_ix::id(), "Enable tower sync vote instruction"),
         (chained_merkle_conflict_duplicate_proofs::id(), "generate duplicate proofs for chained merkle root conflicts"),
         (reward_full_priority_fee::id(), "Reward full priority fee to validators #34731"),
-        (abort_on_invalid_curve::id(), "Abort when elliptic curve syscalls invoked on invalid curve id SIMD-0137")
+        (abort_on_invalid_curve::id(), "Abort when elliptic curve syscalls invoked on invalid curve id SIMD-0137"),
+        (enable_syscall_get_epoch_stake::id(), "Enable syscall: sol_get_epoch_stake #884")
```

Comment:
Just for future understanding, this is supposed to be PR number that introduces the feature gate, not the feature tracking issue?

Reply:
Oh, hmm, I came up with something totally unrelated when I followed that issue number before; I must have typoed 🤦‍♀️ 
We do typically use the PR number here, probably because we often introduce a feature-gate into code before creating the accompanying feature-gate tracking issue. But I see the argument for using the tracking issue number. Feel free to leave this as is.